fifeforspark/utils.py

```diff
@@ -0,0 +1,160 @@
+from pyspark.sql import SparkSession
+import pandas as pd
+import findspark
+import pyspark.sql
+from pyspark.mllib.random import RandomRDDs
+import numpy as np
+from pyspark.sql.types import StructType, StructField, StringType, IntegerType, FloatType
+import random as rn
+import argparse
+
+findspark.init()
+sc = SparkSession.builder.getOrCreate()
```

Comment:
Hey Ed, do we want to initiate the spark session inside the module? It seems like it would be best not to do this.

Reply:
You're absolutely right. Should I transfer the spark session initiation to the __main__.py file?